LIBVmotor0.11/LIBVmotor0.11.ino

```diff
@@ -268,8 +291,43 @@ void loop() {
         }
         break;
     }
+}
+
+void runBPAP(){
+  /*
+  
+  */
+}
+
+bool initHome(){
+  //Initialize the Arms
+  timer = millis();
+  while(true){
+    motor.setPWM(-200);
```

Comment:
These numbers (used in setPWM) should ideally be #defines. This is good both to make it easy to adjust, and easier to read. I.e. setPWM(-CALIBRATION_SPEED_PWM) is more clear than setPWM(-200).

Reply:
I mess up.